words/scales.txt

```diff
@@ -318,3 +318,4 @@ uaru
 vimba
 wahoo
 zebra
+coelacanth
```

Comment:
or do fish go in both files, @catzkorn?

Reply:
If it has a tail and it has scales, it goes in both!

Reply:
Done.